EPI/parse-nested-GrFN.py

```diff
@@ -0,0 +1,124 @@
+import json
+from argparse import ArgumentParser
+
+
+def formatGraph(data): 
+    modelMetadata = {}
+    nodesDict = {}
+    nodes = []
+    edges = []
+
+    for key,value in data.items():
+        if key == 'variables':
+            for item in value: 
+                if 'metadata' in item:
+                    metadata = item['metadata'][0]
+                else:
+                    metadata = {}
+                item['nodeType'] = 'variable'
+                item['metadata'] = metadata
+                nodesDict[item['uid']] = item
+        if key == 'functions':
+            for item in value: 
+                if 'metadata' in item:
+                    metadata = item['metadata'][0]
+                else:
+                    metadata = {} 
+                item['nodeType'] = 'function'
+                item['metadata'] = metadata
+                nodesDict[item['uid']] = item
+        if key == 'hyper_edges':
+            for item in value: 
+                if len(item['inputs']) > 0:
```

Comment:
Do we need this `if len(item['inputs']) > 0`? The body of the following for-loop will not be executed if len(item['inputs']) is 0.

Reply:
For hyper-edges, we sometimes get empty inputs so we don't need to draw that edge. So yes, that's the actual intention, not having that loop to be executed for empty array.s

Reply:
So we should remove this if-statement, correct?

Reply:
Yes, I think so. I'll remove it.